src/Courant/Courant_presets.jl

```diff
@@ -0,0 +1,71 @@
+module Courant
+
+export advective_courant, diffusive_courant, nondiffusive_courant
+
+using LinearAlgebra, StaticArrays
+using ..VariableTemplates
+using ..MoistThermodynamics
+using ..PlanetParameters
+import ..MoistThermodynamics: internal_energy
+using ..SubgridScaleParameters
+using GPUifyLoops
+using CLIMA.Atmos: AtmosModel, soundspeed
+import CLIMA.DGmethods: BalanceLaw, vars_aux, vars_state,
+                        vars_diffusive
+using CLIMA.Mesh.Topologies
+using CLIMA.Mesh.Grids
+using CLIMA.Mesh.Grids: VerticalDirection, HorizontalDirection, EveryDirection
+using Logging
+using Printf
+using LinearAlgebra
+using CLIMA.Atmos
```

Comment:
Please check these -- I believe many are not used.

Reply:
It's likely a few of these might be unnecessary

Reply:
Unused items excluded.